src/library/commissioner_impl.cpp

```diff
@@ -270,6 +271,11 @@ void CommissionerImpl::SetJoinerInfoRequester(JoinerInfoRequester aJoinerInfoReq
     mJoinerInfoRequester = aJoinerInfoRequester;
 }
 
+void CommissionerImpl::SetCommissioningHandler(CommissioningHandler aCommissioningHandler)
+{
+    mCommissioningHandler = aCommissioningHandler;
```

Comment:
I don't see where this `mCommissioningHandler` is used.

Reply:
Yes, the commit is coming.